pkg/drive/event.go

```diff
@@ -345,10 +350,62 @@ func (handler *driveEventHandler) handleUpdate(ctx context.Context, drive *types
 	return nil
 }
 
+func (handler *driveEventHandler) checkDrive(ctx context.Context, drive *types.Drive) error {
+	if drive.Status.Status != directpvtypes.DriveStatusReady {
+		return nil
+	}
+
+	if err := handler.exists(types.GetVolumeRootDir(drive.Status.FSUUID)); err != nil {
+		if errors.Unwrap(err) == syscall.EIO {
+			if uerr := SetIOError(ctx, drive.GetDriveID()); uerr != nil {
+				klog.ErrorS(uerr, "unable to set I/O error", "drive", drive.GetDriveID())
+			}
+		} else {
+			klog.ErrorS(err, "unable to check volume root directory existent", "drive", drive.GetDriveID())
+		}
+
+		return nil
+	}
+
+	if _, err := handler.getDeviceByFSUUID(drive.Status.FSUUID); err != nil {
```

Comment:
we should also revert the state back to "Ready" when the device by FSUUID is found

Reply:
done